lib/clients/Multiplexer/multiplexerclient.py

```diff
@@ -64,6 +64,12 @@ def __init__(self, reactor, parent = None):
         self.d = {}
         self.wmChannels = {}
         self.connect()
+        self._check_window_size()
+
+    def _check_window_size(self):
```

Comment:
docstring clarification about what this function does. e.g. hasattr eliminates the need to have the variable in the config file.